src/screens/CameraScreen/CameraScreen.tsx

```diff
@@ -99,11 +126,21 @@ export const CameraScreen: React.FC = () => {
       existingVideos.push(video);
       await AsyncStorage.setItem(videoKey, JSON.stringify(existingVideos));
     } else {
-      Alert.alert('No video @addMultipleVideos');
+      Toast.show({
+        type: 'error',
+        text1: GENERIC_ERROR_MESSAGE,
+      });
```

Comment:
I think there are way too many generic error messages, but at the same time - the user doesn't need to see what went wrong so maybe it's okay. 

Thinking of maybe if there isn't a camera or something, to ask for permission again.